wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/environment/server/CheEnvironmentValidatorTest.java

```diff
@@ -613,7 +616,7 @@ private static CheServicesEnvironmentImpl createServicesEnv() {
                                                   null,
                                                   emptyList(),
                                                   null);
-        service.setBuild(new CheServiceBuildContextImpl("context", "file", null));
+        service.setBuild(new CheServiceBuildContextImpl("context", "file", null, null));
 
```

Comment:
Add args somewhere in these tests to ensure that they are validated successfully